data/joomla.json

```diff
@@ -1,5 +1,14 @@
 [
   {
+    "title": "[HIGH] Browser information is not filtered properly while saving the session values into the database which leads to a Remote Code Execution vulnerability.",
+    //"edbid": 0,
+    //"osvdbid": 0,
```

Comment:
They're ids for the cve on separate sites exploit-db.com and cvedetails.com
At the time those sites didn't update for the cve, so I left it empty.